lib/msal-react/src/types/Configuration.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export type Configuration = {
```

Comment:
This will conflict with MSAL Browser's configuration, can it be named something more specific?